app/controllers/posts_controller.rb

```diff
@@ -4,17 +4,26 @@ def new
   end
 
   def create
-    @post = Post.create(post_params)
+    # p render plain: params[:post].inspect
+    @post = Post.create!(post_params)
+    #p "images"
+    #p @post.image.attach(params[:image])
```

Comment:
Still got commented code that if not being used, should be removed so the code is more readable.